package.json

```
@@ -170,7 +170,7 @@
     "@babel/register": "^7.5.5",
     "@babel/runtime": "^7.5.5",
     "@changesets/changelog-github": "^0.1.0",
-    "@changesets/cli": "^2.3.0",
+    "@changesets/cli": "^2.11.2",
```

because we're using an unsafe option now

```suggestion
    "@changesets/cli": "2.11.2",
```